subsys/bluetooth/host/audio/endpoint.h

```diff
@@ -62,8 +62,8 @@ struct bt_audio_broadcast_sink {
 	struct bt_audio_capability *cap; /* Capability that accepted the PA sync */
 	struct bt_iso_big *big;
 	struct bt_iso_chan *bis[BROADCAST_SNK_STREAM_CNT];
-	/* The "main" channel used to create the broadcast sink */
-	struct bt_audio_chan *chan;
+	/* The channels used to create the broadcast sink */
+	struct bt_audio_chan *chans;
```

Comment:
I feel I have lost a major point: This is a pointer to an array.  Where is the actual array allocated?  By the user?

Reply:
Yes. We could copy it to here to avoid having the application keep the copy if that gives a better API

Reply:
Fine to keep as is for now, at least.